CMake/ClangCompilers.cmake

```diff
@@ -5,7 +5,7 @@ IF ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.0 )
 ENDIF()
 
 # Enable OpenMP
-SET(ENABLE_OPENMP 0)
+SET(ENABLE_OPENMP 1)
```

Comment:
Considering the increasing use of Clang on HPC, we might need to keep ENABLE_OPENMP as default and specifically protect the Mac if still necessary.